.github/workflows/preview.yml

```diff
@@ -25,6 +25,9 @@ jobs:
         with:
           version: '1.10'
 
+      - name: Load Julia packages from cache
+        uses: julia-actions/cache@v2
+
       # Note: needs resolve() to fix #518
```

Comment:
Once the Quarto `_freeze` cache has been generated for a particular Manifest, the main time it takes to build the docs comes from project instantiation (~ 10 mins) I don't know if this would help it, but I feel like it's worth a shot.